### DISCUSSION TITLE
Why does it need Visual Studio when build vscodium

### DISCUSSION THREAD

**Discussion by xiaochao00** [VSCodium/vscodium#2020] [Q&A] [answered]
When I build vscodium in window platform I get the error:   You need to install the latest version of Visual Studio
![image](https://github.com/user-attachments/assets/0434eea8-4fff-44f6-a18a-9121440c752b)

Does anyone know about this? I just confused about the dependency of vscodium on Visual Studio

**Comment by mcochranca** [comment]
Thats because you need to install the updated or required release version of Codium or VS Code depending on your projects dependencies that can be found  “typically” in your src, root, or project base folder.

**Reply by daiyam** [reply]
No

**Comment by daiyam** [comment, marked as answer]
As the screen tells you, it's build `node-gyp`.

**Reply by xiaochao00** [reply]
yes, I just think this is a simple js/ts program that should be easy to build; As far now I know it`s so hard to build,

**Reply by daiyam** [reply]
No, there are native modules. And if you are building the installers, the `.msi` require some dependencies from Visual Studio (the free version is fine).
